src/common/options.cc

```diff
@@ -3912,7 +3913,8 @@ std::vector<Option> get_global_options() {
 
     Option("osd_recovery_priority", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(5)
-    .set_description(""),
+    .set_description("Priority of recovery in the work queue")
```

Comment:
What is work queue ? which operations go there ?

Reply:
The work queue in this case is a particular OSD's internal queue of things to do that a pool of threads work on.  Things like peering events, recovery, scrub, snap trim, pg delete, and a pg op sent from another osd are on this queue.